qiita_db/metadata_template/test/test_prep_template.py

```diff
@@ -653,73 +653,26 @@ def test_clean_validate_template_error_bad_chars(self):
         self.metadata.index = ['o()xxxx[{::::::::>', 'sample.1', 'sample.3']
```

Comment:
while that does look like a sword, it is not the only thing it resembles at first